sandbox/src/App.tsx

```diff
@@ -88,6 +88,9 @@ function App() {
       <button className="menuButton" onClick={() => connect('bridge')}>Connect using BitBoxBridge</button><br />
       <button className="menuButton" onClick={() => connect('auto')}>Choose automatically</button>
       {err !== undefined && <ErrorNotification message={err.message} code={err.code} onClose={() => setErr(undefined)} /> }
+      <p>
+        This sandbox is using the <a href="https://www.npmjs.com/package/bitbox-api">bitbox-api NPM</a> package.
```

Comment:
nit: link text is "bitbox-api NPM" It would be nice if link text was "BitBox sandbox".  How about:

The bitbox-api npm can be used in the BitBox sandbox.

Reply:
omg,  all good  lol.  no nit 🙈